stalker/code/objects/items/weapons/loot.dm

```diff
@@ -3,39 +3,38 @@
 
 /obj/item/stalker/loot/dog_tail
 	name = "dog tail"
-	desc = "Устойчивость кожи мутировавших собак к химическому и электрическому воздействию давно обратила на себя внимание учёных. Из-за отсутствия жировых отложений хвосты собак наиболее подходят для лабораторных исследований"
 	eng_desc = "The resistance of mutant dogs' skin to electricity and chemical agents has long been the subject of scientific attention. Since the animal's tail is almost completely free of fat tissue, it is especially suited for lab experimentation."
 	icon_state = "pes"
 
 /obj/item/stalker/loot/flesh_eye
 	name = "flesh eye"
-	desc = "После мутаций увеличенный глаз Плоти приобрёл ряд свойств, самым полезным из которых является способность к регенерации хрусталика. Детали этого механизма точно не известны, но это открытие обещает революцию в медицине. Вы можете внести свой вклад в науку, продав глаз плоти учёным для исследований."
 	eng_desc = "The mutated, swollen eye of the Flesh acquired a number of strange characteristics, the most useful of which is the ability to regenerate the lens. The specifics of the process are unknown, although such discovery would most certainly revolutionize modern medicine. You can play your part in scientific development by selling Flesh eyes to scientists for their experiments."
 	icon_state = "plot"
 
 /obj/item/stalker/loot/boar_leg
 	name = "boar's hoof"
-	desc = "Копыто кабана используется учёными для изготовления ряда лекарств, самым распространённым из которых является противорадиационная сыворотка."
 	eng_desc = "The hoof of a wild boar is used by scientists for manufacturing of some medicines, the most widespread of which are anti radiation drugs."
 	icon_state = "kaban"
 
 /obj/item/stalker/loot/snork_leg
 	name = "snork leg"
-	desc = "The resistance of mutant dogs' skin to electricity and chemical agents has long been the subject of scientific attention. Since the animal's tail is almost completely free of fat tissue, it is especially suited for lab experimentation."
 	eng_desc = "Snork feet contain a huge number of highly elastic tendons. This explains the beast's ability to make those unbelievably long jumps."
 	icon_state = "snork_leg"
 
 /obj/item/stalker/loot/pseudo_tail
 	name = "pseudo-dog tail"
-	desc = "Устойчивость кожи мутировавших собак к химическому и электрическому воздействию давно обратила на себя внимание учёных. Из-за отсутствия жировых отложений хвосты собак наиболее подходят для лабораторных исследований."
+	eng_desc = "Устойчивость кожи мутировавших собак к химическому и электрическому воздействию давно обратила на себя внимание учёных. Из-за отсутствия жировых отложений хвосты собак наиболее подходят для лабораторных исследований."
```

Comment:
Still in Russian?

Reply:
Looks like this still needs to be translated, great job otherwise!